packages/html/src/calendar/tests/calendar-infinite-long-names.tsx

```diff
@@ -0,0 +1,25 @@
+import { CalendarInfiniteNormal } from '../index';
+
+export default () => (
+    <>
+        <div id="test-area" className="k-grid">
```

Comment:
Typo in `test-area` element, the className should be `k-d-grid`. 
`k-grid` breaks the calendar cells in the visual tests.